x-pack/plugins/observability/public/pages/alert_details/components/header_actions.tsx

```diff
@@ -0,0 +1,186 @@
+  } = useKibana<ObservabilityAppServices>().services;
+
+  const { rule, reloadRule } = useFetchRule({
+    http,
+    ruleId: alert?.fields[ALERT_RULE_UUID] || '',
+  });
+
+  const [isPopoverOpen, setIsPopoverOpen] = useState<boolean>(false);
+  const [ruleConditionsFlyoutOpen, setRuleConditionsFlyoutOpen] = useState<boolean>(false);
+  const [snoozeModalOpen, setSnoozeModalOpen] = useState<boolean>(false);
+
+  const selectCaseModal = getUseCasesAddToExistingCaseModal();
+
+  const handleTogglePopover = () => setIsPopoverOpen(!isPopoverOpen);
+  const handleClosePopover = () => setIsPopoverOpen(false);
+
+  const attachments: CaseAttachmentsWithoutOwner =
+    alert && rule
+      ? [
+          {
+            alertId: alert?.fields[ALERT_UUID] || '',
+            index: '.internal.alerts-observability.metrics.alerts-*',
+            rule: {
+              id: rule.id,
+              name: rule.name,
+            },
+            type: CommentType.alert,
+          },
+        ]
+      : [];
+
+  const handleAddToCase = () => {
+    setIsPopoverOpen(false);
+    selectCaseModal.open({ attachments });
+  };
+
+  const handleViewRuleConditions = () => {
+    setIsPopoverOpen(false);
+    setRuleConditionsFlyoutOpen(true);
+  };
+
+  const handleOpenSnoozeModal = () => {
+    setIsPopoverOpen(false);
+    setSnoozeModalOpen(true);
+  };
+
+  const handleOpenInExternalApp = () => {
+    if (alert) {
+      externalConnector?.onViewInExternalApp(alert.fields[ALERT_UUID]);
+    }
+  };
+
+  return (
+    <>
+      <EuiPopover
+        isOpen={isPopoverOpen}
+        closePopover={handleClosePopover}
+        button={
+          <EuiButton
+            fill
+            iconType="arrowDown"
+            iconSide="right"
+            onClick={handleTogglePopover}
+            data-test-subj="alert-details-header-actions-menu-button"
+          >
+            {i18n.translate('xpack.observability.alertDetails.actionsButtonLabel', {
+              defaultMessage: 'Actions',
+            })}
+          </EuiButton>
+        }
+      >
+        <EuiFlexGroup direction="column" alignItems="flexStart">
+          <EuiButtonEmpty
+            size="s"
+            color="text"
+            onClick={handleAddToCase}
+            data-test-subj="add-to-case-button"
+          >
+            <EuiText size="s">
+              {i18n.translate('xpack.observability.alertDetails.addToCase', {
+                defaultMessage: 'Add to case',
+              })}
+            </EuiText>
+          </EuiButtonEmpty>
+
+          <EuiButtonEmpty
+            size="s"
+            color="text"
+            disabled={!alert?.fields[ALERT_RULE_UUID]}
+            onClick={handleViewRuleConditions}
+            data-test-subj="view-rule-conditions-button"
+          >
+            <EuiText size="s">
+              {i18n.translate('xpack.observability.alertDetails.viewRuleConditons', {
+                defaultMessage: 'View rule conditions',
```

Comment:
I want to confirm if the Actions list names are as per design because in the Alerts table, we have `View rule details`. Also, there is `Go to APM` for APM rule alerts (same for other rules). Is this option not to be added in Actions? 

From Figma design - 
<img width="210" alt="Screenshot 2022-09-30 at 10 54 09" src="https://user-images.githubusercontent.com/69037875/193234102-23726236-95b7-481b-b5af-c21852d11433.png">

Reply:
I’ve discussed this with @maciejforcone. Could you weigh in here please?

Reply:
@benakansara @CoenWarmer let's add this action on top, this is from latest consultations I had with Vinay. Also to your question about the "active" badge - status still should be active, user already received notification (before it was snoozed), and alert didn't recover.

Reply:
@maciejforcone It's not clear to me yet. Could you please clarify -
1) Are we going with `View rule conditions` in place of `View rule details`?
2) Do we have `View in app` or `Go to APM` under Actions?

Reply:
And just to make it more challenging regarding the `View rule conditions` 😅 , this option in the **Rule Details** page is `Edit rule` and it opens the same flyout that the Alert Details open here. Shouldn't we align the title here?

Reply:
Just spoke to @maciejforcone and no, we do not want to align these titles at this time.